From 5510f630c353aba96e8289062d5375aa6074c9c2 Mon Sep 17 00:00:00 2001
From: Zhao Wei Liew <zhaoweiliew@gmail.com>
Date: Sat, 4 Feb 2017 15:09:31 +0800
Subject: [PATCH 149/214] www: config: Correct LineageOS name

There isn't supposed to be a space between 'Lineage' and 'OS'.

Change-Id: Icb2b68afea15d4a174c4f7da32bb4b8c7b6260f4
---
 _config.yml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/_config.yml b/_config.yml
index 4196c48..69d2362 100644
--- a/_config.yml
+++ b/_config.yml
@@ -3,10 +3,10 @@
 #
 
 # Name of your site (displayed in the header)
-name: Lineage OS
+name: LineageOS
 
 # Short bio or description (displayed in the header)
-description: Lineage OS Android Distribution
+description: LineageOS Android Distribution
 
 
 #
-- 
2.17.1

